custom/icds_reports/const.py

```diff
@@ -111,6 +111,8 @@ class MapColors(object):
 AGG_THR_V2_TABLE = 'icds_dashboard_thr_v2'
 AGG_DASHBOARD_ACTIVITY = 'icds_dashboard_user_activity'
 AGG_ADOLESCENT_GIRLS_REGISTRATION_TABLE = 'icds_dashboard_adolescent_girls_registration'
+AGG_GOV_VHND_TABLE = 'icds_dashboard_gov_vhnd_forms'
```

Comment:
is this still needed after you shifted the vhnd to gov dashboard table?

Reply:
removed